Content.Server/Medical/SuitSensors/SuitSensorSystem.cs

```diff
@@ -339,6 +340,11 @@ public void SetSensor(EntityUid uid, SuitSensorMode mode, EntityUid? userUid = n
         var isAlive = false;
         if (EntityManager.TryGetComponent(sensor.User.Value, out MobStateComponent? mobState))
             isAlive = !_mobStateSystem.IsDead(sensor.User.Value, mobState);
+        
+        // Floofstation - get IPC battery status
+        var isDischarged = false;
+        if (EntityManager.TryGetComponent(sensor.User.Value, out SiliconComponent? SiliconComp) && (SiliconComp.ChargeState == 0))
+            isDischarged = true;
```

Comment:
No need for an if expression, and use EntitySystem.TryComp instead of EntityManager.TryGetComponent:
```suggestion
        var isDischarged = TryComp(sensor.User.Value, out SiliconComponent? SiliconComp) && (SiliconComp.ChargeState == 0);
```

Reply:
Thanks, that's much cleaner. What's the benefit of using TryComp over TryGetComponent? Is it just that it's native since this is an EntitySystem? I was mostly just referencing the isAlive check above it.

Reply:
It's just more concise, it still calls EntMan.TryGetComponent under the hood